## Florida Vacation


In [134]:
# Disney World Vacation

# Family info
family_of_four = 4

# Disney World Price
def disney_world_cost():
    disney_world_hotel = 260 * 0.72# Cost of contemporary 670 * 0.72# cost of Animal 540 * 0.72# cost of coronado 260*.72 
    hotel_nights = 5
    disney_world_tickets = 290 * family_of_four
    hotel_cost = disney_world_hotel * hotel_nights
    total_cost = disney_world_tickets + hotel_cost
    return total_cost

In [135]:

# Universal Price
def universal_cost():
    universal_hotel = 404
    hotel_nights = 3
    universal_tickets = 253 * family_of_four
    hotel_cost = universal_hotel * hotel_nights
    total_cost = universal_tickets + hotel_cost
    return total_cost



In [136]:
# Flights
def plane_tickets():
    ticket_price = 295
    baggage = 35 * 2
    return (ticket_price + baggage) * family_of_four



In [137]:
# Food costs
def food_cost():
    food_per_person = 80
    number_of_days = 9
    return food_per_person * number_of_days * family_of_four



In [138]:
# Rental car
def rental_car():
    rental_car = 670 #741 for ten days #670 for 9 days 
    gas = 100
    return rental_car + gas



In [139]:
def daytona_tickets():
    roar_before = 0 #20 * 4
    rolex_24 = 60 * 4
    return roar_before + rolex_24

In [140]:
# Total trip cost
def total_trip_cost():
    disney_cost = disney_world_cost() 
    uni_cost = universal_cost() 
    plane_cost = plane_tickets() 
    food = food_cost()  
    car = rental_car()
    rolex = daytona_tickets()
    return disney_cost + uni_cost + plane_cost + food + car + rolex


In [141]:
# Travel Cost
def travel_cost():
    plane_cost = plane_tickets()
    car_cost = rental_car()
    return plane_cost + car_cost

In [142]:

print("The total cost of your trip is: ",  total_trip_cost())
print("The total cost of Disney World is: ", disney_world_cost())
print("The total cost of Universal is: ", universal_cost())
print("The total cost of Travel is: ", travel_cost())
print("The total cost of food is: ", food_cost())
print("The total cost of Daytona Race is: ", daytona_tickets())

The total cost of your trip is:  9670.0
The total cost of Disney World is:  2096.0
The total cost of Universal is:  2224
The total cost of Travel is:  2230
The total cost of food is:  2880
The total cost of Daytona Race is:  240


### Cost for Ten Days with stay at Contemporary and Hard Rock:
- The total cost of your trip is:  12021.0
- The total cost of Disney World is:  3572.0
- The total cost of Universal is:  2628
- The total cost of Travel is:  2301
- The total cost of food is:  3200
- The total cost of Daytona Race is:  320

### Cost for Ten Days with stay at Animal Kingdom and Hard Rock:
- The total cost of your trip is:  11553.0
- The total cost of Disney World is:  3104.0
- The total cost of Universal is:  2628
- The total cost of Travel is:  2301
- The total cost of food is:  3200
- The total cost of Daytona Race is:  320

### Cost for Ten Days with stay at Corornado and Hard Rock:
- The total cost of your trip is:  10545.0
- The total cost of Disney World is:  2096.0
- The total cost of Universal is:  2628
- The total cost of Travel is:  2301
- The total cost of food is:  3200
- The total cost of Daytona Race is:  320

### Cost for Nine Days with stay at Coronado and Hard Rock:
- The total cost of your trip is:  9670.0
- The total cost of Disney World is:  2096.0
- The total cost of Universal is:  2224
- The total cost of Travel is:  2230
- The total cost of food is:  2880
- The total cost of Daytona Race is:  240